### ISSUE TITLE
Incorrect handling of US International keyboard layout (or of Typematrix keyboard ?)

### ISSUE THREAD

**Issue by flure** [defold/editor2-issues#2011]
I'm using a qwerty Typematrix keyboard, using US International layout on Linux Debian. I use it so I can type all the french characters.
The issue is that since I've switched from Editor 1 to Editor 2, I can't type the characters `'` or `"` in the code editor. (I can in any other software though).
I suspect the code editor doesn't handle the US International layout **or** the Typematrix keyboard correctly.
This is impacting my productivity a lot, I always have to switch my keyboard layout everytime I need to type any of those two characters.
There may be other characters that don't work, but I haven't found them yet.

**Comment by erikangelin-king** [comment]
Can you type the characters in for instance the edit fields in the property panel?

**Comment by flure** [comment]
Yes, I can type those characters in edit fields with no problem, I just checked. Apparently the problem is only in the code editor.
